src/components/Header/Menu.tsx

```diff
@@ -60,8 +60,8 @@ export const Menu: FunctionComponent<MenuProps> = ({ isOpen, onClick }) => {
                 name: "Further information",
                 links: [
                   { name: "Quality", url: "/quality" },
-                  { name: "Supported Platforms", url: "/supported_platforms" },
-                  { name: "Get involved!", url: "/getinvolved" },
+                  { name: "Supported Platforms", url: "/supported-platforms" },
```

Comment:
If we change URLs we're going to want to provide redirects. There's no point in messing with the SEO if we don't need to. 

This can be configured in gatsby-node.js. Here are some examples: https://github.com/gatsbyjs/gatsby/blob/master/examples/using-redirects/gatsby-node.js

I'm not sure how to do an actual 301/302 response from a page in GitHub pages (I've configured it in S3 before), but we would probably be able to get away with a meta refresh tag, which I think it what it does by default when you `createRedirect`.

Reply:
added redirects